scripts/open_url_in_instance.sh

```diff
@@ -6,13 +6,10 @@ _url="$1"
 _qb_version='0.10.1'
 _proto_version=1
 _ipc_socket="${XDG_RUNTIME_DIR}/qutebrowser/ipc-$(echo -n "$USER" | md5sum | cut -d' ' -f1)"
+_qute_bin="/usr/bin/qutebrowser"
 
-if [[ -e "${_ipc_socket}" ]]; then
-	exec printf '{"args": ["%s"], "target_arg": null, "version": "%s", "protocol_version": %d, "cwd": "%s"}\n' \
-				"${_url}" \
-				"${_qb_version}" \
-				"${_proto_version}" \
-				"${PWD}" | socat - UNIX-CONNECT:"${_ipc_socket}"
-else
-    exec /usr/bin/qutebrowser --backend webengine "$@"
-fi
+exec printf '{"args": ["%s"], "target_arg": null, "version": "%s", "protocol_version": %d, "cwd": "%s"}\n' \
```

Comment:
Hmm, trying with `exec false || gedit` I never see anything launched, as `exec false` is done and the shell doesn't exist anymore.

So I don't think this will work either? I think it's fine to just drop the exec.

Reply:
That's weird, I'm using this with the `exec` in atm., and it's working fine. Though I agree, thinking about it, it probably shouldn't be working. 🤔
Oh well, I'll just drop it, shouldn't hurt anything.